package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "private": true,
+  "workspaces": [
+    "clients/tabby-agent",
+    "clients/vscode",
+    "clients/vim"
+  ]
+}
```

Comment:
Add a workspace package.json in root dir.  
Is it better to move this file to ./clients?

Reply:
in root is fine